docs/integrations/github/org.md

```diff
@@ -91,6 +93,7 @@ Directly under the `githubOrg` is a list of configurations, each entry is a stru
 - `id`: A stable id for this provider. Entities from this provider will be associated with this ID, so you should take care not to change it over time since that may lead to orphaned entities and/or conflicts.
 - `githubUrl`: The target that this provider should consume
 - `orgs` (optional): The list of the GitHub orgs to consume. If you only list a single org the generated group entities will use the `default` namespace, otherwise they will use the org name as the namespace. By default the provider will consume all accessible orgs on the given GitHub instance (support for GitHub App integration only).
+- `alwaysUseDefaultNamespace` (optional): If set to `true`, the provider will always use the `default` namespace for the generated group entities. Groups with the same name across different orgs will be considered the same group. Defaults to `false`.
```

Comment:
I know this may seem obvious but we should call out what default is for Users

Reply:
You're right - actually I can't answer 🙈 
My assumption is, that this config option also applies to users, isn't it?

Reply:
It only applies to groups. It's actually a bit of an internal thing for the provider in the new backend system, previously you'd use the plan provider instead of the multi one. I think it's good as is 👍

Reply:
The reason I've mentioned it is that I've seen this come up where people intentionally want users in a different namespace. Was just wanting to avoid the confusion for that case but I'm good with not adding it 👍

Reply:
Got it - thanks for the explanation 🙏🏼